src/Compiler/FSComp.txt

```diff
@@ -1538,6 +1538,7 @@ tcPassingWithoutNullToNonNullQuickAP,"You can remove this |NonNullQuick| pattern
 tcPassingWithoutNullTononNullFunction,"You can remove this `nonNull` assertion."
 3263,tcNullableToStringOverride,"With nullness checking enabled, overrides of .ToString() method must return a non-nullable string. You can handle potential nulls via the built-in string function."
 3264,tcDowncastFromNullableToWithoutNull,"Nullness warning: Downcasting from '%s' into '%s' can introduce unexpected null values. Cast to '%s|null' instead or handle the null before downcasting."
+3265,tcDisallowedNullableApplication,"Application of method '%s' attempted to create a nullable type ('T | null) for '%s'. Nullness warnings won't be reported correctly for such types."
```

Comment:
Maybe make it more explicit why it's a problem.

```suggestion
3265,tcDisallowedNullableApplication,"Application of method '%s' attempted to create a nullable type ('T | null) for '%s', which does not support nullability. Nullness warnings won't be reported correctly for this type."
```

Reply:
This might be confusing, since `System.Nullable` exists, and people think about it as well when they thing "nullabillity" in .NET.

Reply:
True.
I will go with the current message.

It is indeed likely that most current examples of this warning will come via structs.